force-app/main/default/classes/DMLOperations.cls

```diff
@@ -162,12 +225,21 @@ public with sharing class DMLOperations {
 		// Query for the Account
 		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
 
-		// if the Account exists, update the description
-
-		// otherwise, create a new Account and set the description
+       Account account;
+       
+        //if the account exists, update the description, else create a new one
+        if (!accounts.isEmpty()){
```

Comment:
Here you queried into a List and then used isEmpty. Can you look at some of the ones above where that might be useful?